nsv13/code/modules/overmap/overmap.dm

```diff
@@ -50,7 +50,7 @@
 	var/desired_angle = null // set by pilot moving his mouse
 	var/angular_velocity = 0 // degrees per second
 	var/max_angular_acceleration = 180 // in degrees per second per second
-	var/speed_limit = 5 //Stops ships from going too damn fast. This can be overridden by things like fighters launching from tubes, so it's not a const.
+	var/speed_limit = 10 //Stops ships from going too damn fast. This can be overridden by things like fighters launching from tubes, so it's not a const.
```

Comment:
This is going to make AI ships extremely annoying to hunt down.

Reply:
yep! although at the same time it may give an incentive for people to build meta around the gravity stuff. this was the hardest part of balancing for me, figuring out how to allow fast velocities closer to planets for slingshots, but preventing this stuff as they fly away. I need to look into this more. the system right now is kinda weird and limits gravity if you're going directly away from the source of a gravity, to prevent getting stuck, and removing that may help.. but now you're stuck and such.